lndc/conn.go

```diff
@@ -72,9 +72,24 @@ func parseAdr(netAddress string) (string, string, error) {
 	} else if colonCount >= 5 {
 		conMode = "tcp6"
 		return netAddress, conMode, nil
-	} else {
-		return "", "", fmt.Errorf("Invalid ip")
+	} else if colonCount == 1 {
```

Comment:
Nice, I wish we could remove the ugly method of checking ips though, there must be a better way..